irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -23,71 +25,134 @@ namespace iroha {
     PostgresWsvCommand::PostgresWsvCommand(pqxx::nontransaction &transaction)
         : transaction_(transaction),
           log_(logger::log("PostgresWsvCommand")),
-          execute_{makeExecute(transaction_, log_)} {}
+          execute_{makeExecuteResult(transaction_, log_)} {}
+
+    WsvCommandResult PostgresWsvCommand::insertRole(
+        const std::string &role_name) {
+      auto result = execute_("INSERT INTO role(role_id) VALUES ("
+                             + transaction_.quote(role_name) + ");");
+
+      auto message_gen = [&] {
+        return (boost::format("failed to insert role: '%s'") % role_name).str();
```

Comment:
Maybe use capital letters for errors?

Reply:
They are intended to be part of larger message (with command name, transaction info, etc) so capitalizing might look out of place.